Makefile

```diff
@@ -38,8 +38,8 @@ up: vendor start config/config.toml ## Set up the development environment
 
 .PHONY: down
 down: clean ## Destroy the development environment
-	docker-compose down
-	rm -rf .docker/
+	uid=$(shell id -u) gid=$(shell id -g) docker-compose down
+	@ if [[ "$$OSTYPE" == "linux-gnu" ]]; then sudo rm -rf .docker/; else rm -rf .docker/; fi
```

Comment:
Isn't this resolved by the user setting?

Reply:
Unfortunately, it was not resolved for me which is not so nice. Even though I created the folders with my Linux user, the folders under `.docker/volumes/vault/file` were created by root. That's why I could not delete the whole folder without `sudo`.

 As far as I know, the `vault` container creates these files, so I've also tried to launch that container as well with my local user to solve this issue. In that case, the `vault` container exited with a `permission denied` error. That's why I simply solved the issue with `sudo rm`, but let me know if you have another idea!

Reply:
Unexisting external mount points are created by the Docker daemon implicitly, which runs as root, so on Linux the implicitly created folders will be owned by root.

Reply:
But the mount points are created `make start`...anyway, I guess some processes (eg mysql) could run with different users, so we probably have limited options here.

Reply:
@sagikazarmark I'm not sure I understand, are you suggesting some changes with your last comment? For me, at least, it seemed to work.

Reply:
No-no, it's okay as it is now